src/gui/editors/AutomationEditor.cpp

```diff
@@ -862,6 +925,56 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 				}
 			break;
 			}
+			case EditMode::EditTangents:
+			{
+				// If we moved the mouse past the beginning correct the position in ticks
+				posTicks = qMax(posTicks, 0);
```

Comment:
```suggestion
				posTicks = std::max(posTicks, 0);
```
This isn't a necessary change since this is gui code, so it's up to you.